### DISCUSSION TITLE
OpenTOPAS Discussions

### DISCUSSION THREAD

**Discussion by jschuemann** [OpenTOPAS/OpenTOPAS#11] [Announcements]
We want you to know that we aim to answer your questions here. In this transition phase, we may be slower since we are still working on improving your experience of this resource. Please help us by answering questions where you know the answer.

**Comment by jschuemann** [comment]
Great to see you here.

**Comment by mforrest-tmu** [comment]
Why is this fork here?  Why not continue contributing code to the topasmc repo?

As a tech, this complicates my life as I'll now probably have to maintain patches for two separate code bases.

**Comment by JoseRamosMendez** [comment]
Hi there! 

We know this can be complicated at first. But we, as Collaboration, decided to set up a new repository, not a fork, which provides more openness to the users and allows us an inter-institutional decision-making workflow rather than a single-person decision way. 

We recommend using this upgraded version. it has been tested and passed all the regression tests covering physics models and end-to-end tests, compatible with a more recent Geant4 version.
